library/swift/src/StreamClientBuilder.swift

```diff
@@ -107,6 +108,17 @@ public final class StreamClientBuilder: NSObject {
     return self
   }
 
+  /// Add HTTP filter chain for requests and responses (i.e. streams) managed by this client.
+  ///
+  /// - parameter filterChain: Filter chain to be invoked for streams.
+  ///
+  /// - returns: This builder.
+  @discardableResult
+  public func addFilterChain(_ filterChain: [RequestFilter]) -> StreamClientBuilder {
```

Comment:
This should probably take an individual filter, which gets appended to the underlying chain (rather than replacing)

Reply:
Also what are your thoughts on how inter-op ordering will work here with regard to C++ filters?

Reply:
We have a few options:
1. platform filters always come first on the request path / last on the response path - straightforward and what we'll do initially.
2. platform filters can be appended _or_ prepended - also pretty straightforward.
3. represent c++ filters as part of a platform-exposed chain that can be interacted with for arbitrary ordering - more complicated, and something to consider later, if necessary. possible, though.

Reply:
I'm up for doing #1 for MVP 👍